ui-tests/src/test/java/io/syndesis/qe/steps/other/DocsVerificationSteps.java

```diff
@@ -0,0 +1,50 @@
+package io.syndesis.qe.steps.other;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.WebDriverRunner.url;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+import com.codeborne.selenide.Selenide;
+
+import java.util.HashMap;
+import java.util.concurrent.TimeUnit;
+
+import cucumber.api.java.en.Then;
+
+public class DocsVerificationSteps {
+    private HashMap<String, String> versions = new HashMap<>();
+    private static String versionOnAboutPage;
+
+    public DocsVerificationSteps() {
+        versions.put("1.9", "7.6");
+    }
+
+    @Then("^check version in about page$")
+    public void checkVersion() {
+        final By SYNDESIS_VERSION = By.cssSelector("[data-testid=\"about-modal-content-version-list-item\"]");
+        assertThat($(SYNDESIS_VERSION).shouldBe(visible).getText())
+            .isNotEmpty();
+        versionOnAboutPage = $(SYNDESIS_VERSION).getText();
+    }
+
+    @Then("verify whether the docs has right version")
+    public void verifyDocsVersion() {
+        Selenide.switchTo().window(1);
+        FirefoxDriver driver = new FirefoxDriver();
+        driver.manage().timeouts().implicitlyWait(2, TimeUnit.SECONDS);
+        assertThat(versions.get(versionOnAboutPage.substring(0, 3))).isEqualTo(url().split("/")[6]);
+
+        if (!(versionOnAboutPage.contains("redhat"))) {
+            final By DOCS_VERSION = By.cssSelector(".productnumber");
+            assertThat($(DOCS_VERSION).shouldBe(visible).getText())
+                .isNotEmpty();
+
+            assertThat(versions.get(versionOnAboutPage)).isEqualTo($(DOCS_VERSION).getText());
+        }
```

Comment:
this will fail for the whole release cycle, as the documentation isn't publicly available on the given URL until the release is done.

I would say that it is enough to check the URL if it links to correct version